espnet2/bin/asr_inference.py

```diff
@@ -14,6 +14,9 @@
 from typeguard import check_return_type
 from typing import List
 
+from espnet.nets.beam_search_transducer_espnet2 import BeamSearchTransducerESPnet2
+from espnet.nets.beam_search_transducer_espnet2 import Hypothesis as RNNTHypothesis
+
```

Comment:
Should we use the name of `RNNT` in RNNTHypothesis? TransducerHypothesis is more general?

Reply:
I've gone for Transducer for now!